### PR TITLE
Feature/version up

### PR DESCRIPTION
**Pull request by minsulee2**

### CODE REVIEW

**Comment by minsulee2** [comment]
@liayoo As discussed in the afternoon, semver has been adopted! PTAL

File: client/index.js [hunk 1]
```diff
@@ -31,12 +31,14 @@ if (!fs.existsSync(PROTOCOL_VERSIONS)) {
   throw Error('Missing protocol versions file: ' + PROTOCOL_VERSIONS);
 }
 const VERSION_LIST = JSON.parse(fs.readFileSync(PROTOCOL_VERSIONS));
-if (!VERSION_LIST[CURRENT_PROTOCOL_VERSION]) {
+const MAJOR_MINOR_VERSION =
+    `${semver.major(CURRENT_PROTOCOL_VERSION)}.${semver.minor(CURRENT_PROTOCOL_VERSION)}`;
+if (semver.valid(MAJOR_MINOR_VERSION) && !VERSION_LIST[MAJOR_MINOR_VERSION]) {
```
Comment (liayoo) [inline client/index.js:36]: Should we throw an error for the `!semver.valid(..)` case?

**Review by liayoo — APPROVED** [review]
LGTM with a comment. Thanks for these updates!
Could you please merge after PR #121 is merged?

File: client/index.js (hunk 1 above)
Comment (minsulee2) [reply client/index.js]: Separated em! Thanks for the comment!

Comment (liayoo) [reply client/index.js]: Thanks! 👍

**Comment by minsulee2** [comment]
Thanks for being patient!! :)

**Pull request merged**